runcards/scaling/1d_sweepers.yml

```
@@ -0,0 +1,292 @@
+platform: iqm5q
```

Here maybe we should put
```suggestion
platform: dummy
```
to make it neutral, like the other file.

Also, I am not sure if we really need the folder as it only contains one file. Maybe we can just rename it `scaling_benchmarks.yml` or something like that and leave it under runcards.

---

I left just in case we go for a 2d_sweepers thing in the future, but yeah right now it's a bit pointless.

---

Sure, that is fine with me. Thanks for the updates.

---

I will add the dummy as well when I can push